### PR TITLE
Reused nexus/core/schema from nexus commons as main schema context

### LINKED ISSUE #152: Reuse the SHACL context defined in nexus-commons
Currently the used SHACL context is defined in [nexus-prov](https://github.com/BlueBrain/nexus-prov) v0.1.1. 
The SHACL context defined in [nexus-commons](https://github.com/BlueBrain/nexus-commons) should be used.

### PR DESCRIPTION
**Pull request by MFSY**
- Used nxv:Schema as schema type
- Fixes #152 #158 #160

### CODE REVIEW

File: modules/bbp-experiment/src/main/resources/schemas/bbp/experiment/patchedcell/v0.1.0.json [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
     }
   ],
-
-  "@type" : "owl:Ontology",
-  "imports" : [ "{{base}}/schemas/bbp/core/entity/v0.1.0",
-                "{{base}}/schemas/bbp/core/quantitativevalue/v0.1.0",
-                "{{base}}/schemas/bbp/core/typedlabeledontologyterm/v0.1.0",
-                "{{base}}/schemas/bbp/core/brainlocation/v0.1.0"],
-  "shapes" : [
+  "@type": "owl:Ontology",
```
Comment (annakristinkaufmann) [inline modules/bbp-experiment/src/main/resources/schemas/bbp/experiment/patchedcell/v0.1.0.json]: should this be changed to "nxv:Schema"?

File: modules/bbp-experiment/src/main/resources/schemas/bbp/experiment/patchedslice/v0.1.0.json [hunk 2]
```diff
@@ -1,20 +1,21 @@
...
+      "bbptlotsh": "{{base}}/schemas/bbp/core/typedlabeledontologyterm/v0.1.0/shapes/",
+      "this": "{{base}}/schemas/bbp/experiment/patchedslice/v0.1.0/shapes/"
     }
   ],
-
-  "@type" : "owl:Ontology",
-  "imports" : [ "{{base}}/schemas/bbp/core/entity/v0.1.0",
-                "{{base}}/schemas/bbp/core/typedlabeledontologyterm/v0.1.0"],
-  "shapes" : [
+  "@type": "owl:Ontology",
```
Comment (annakristinkaufmann) [inline modules/bbp-experiment/src/main/resources/schemas/bbp/experiment/patchedslice/v0.1.0.json]: should this be changed to "nxv:Schema"?

File: modules/bbp-experiment/src/main/resources/schemas/bbp/experiment/protocol/v0.1.0.json [hunk 3]
```diff
@@ -29,14 +28,15 @@
           "path": "schema:identifier",
           "name": "Identifier",
           "description": "The identifier of protocol such as a DOI, ISBN,...",
-          "node": "this:IdentifierShape",
+          "datatype": "xsd:anyURI",
```
Comment (annakristinkaufmann) [inline modules/bbp-experiment/src/main/resources/schemas/bbp/experiment/protocol/v0.1.0.json]: Not every protocol might have a URI (e.g. if we only get a pdf of the protocol). Maybe we should remove the minCount on the identifier property?

File: modules/bbp-experiment/src/main/resources/schemas/bbp/experiment/subject/catalog-v001.xml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (annakristinkaufmann) [inline modules/bbp-experiment/src/main/resources/schemas/bbp/experiment/subject/catalog-v001.xml]: What is this file?

File: modules/bbp-experiment/src/main/resources/schemas/bbp/experiment/subject/v0.1.0.json.ttl [hunk 5]
```diff
@@ -0,0 +1,44 @@
+<https://bbp-nexus.epfl.ch/vocabs/bbp/neurosciencegraph/core/v0.1.0/transgenic>
```
Comment (annakristinkaufmann) [inline modules/bbp-experiment/src/main/resources/schemas/bbp/experiment/subject/v0.1.0.json.ttl]: what is this file?

File: modules/bbp-simulation/src/main/resources/schemas/bbp/simulation/circuit/1 [hunk 6]
```diff
@@ -0,0 +1,24 @@
+{
```
Comment (annakristinkaufmann) [inline modules/bbp-simulation/src/main/resources/schemas/bbp/simulation/circuit/1]: what is this file?

File: modules/bbp-experiment/src/main/resources/schemas/bbp/experiment/patchedcell/v0.1.0.json (hunk 1 above)
Comment (MFSY) [reply modules/bbp-experiment/src/main/resources/schemas/bbp/experiment/patchedcell/v0.1.0.json]: Yes. Nice catch.

File: modules/bbp-experiment/src/main/resources/schemas/bbp/experiment/patchedslice/v0.1.0.json (hunk 2 above)
Comment (MFSY) [reply modules/bbp-experiment/src/main/resources/schemas/bbp/experiment/patchedslice/v0.1.0.json]: Done

File: modules/bbp-experiment/src/main/resources/schemas/bbp/experiment/protocol/v0.1.0.json (hunk 3 above)
Comment (MFSY) [reply modules/bbp-experiment/src/main/resources/schemas/bbp/experiment/protocol/v0.1.0.json]: Okay. Removed mincount and maxcount.

File: modules/bbp-experiment/src/main/resources/schemas/bbp/experiment/subject/v0.1.0.json.ttl (hunk 5 above)
Comment (MFSY) [reply modules/bbp-experiment/src/main/resources/schemas/bbp/experiment/subject/v0.1.0.json.ttl]: the joy of using protege. I removed it

File: modules/bbp-simulation/src/main/resources/schemas/bbp/simulation/circuit/1 (hunk 6 above)
Comment (MFSY) [reply modules/bbp-simulation/src/main/resources/schemas/bbp/simulation/circuit/1]: I Removed it.

**Pull request merged**